config/gcpmanager/gcp-manager.yaml

```diff
@@ -24,7 +24,8 @@ spec:
       containers:
         - name: gcp-manager
           image: us-central1-docker.pkg.dev/substratus-ai-001/substratus/gcpmanager:latest
-          # image: substratusai/gcp-manager:latest
+          # image: substratusai/gcp-manager:main
```

Comment:
I'd like to keep `image: substratusai/gcp-manager:main` or `image: substratusai/gcp-manager:latest` as live here but skaffold doesn't seem to replace the image unless it matches (I think)